### PR TITLE
Add content-security-policy header

### PR DESCRIPTION
**Pull request by Rosa-Fox**
- Increase security by helping to mitigate cross-site scripting and data injection attacks.
- A CSP compatible browser will only execute scripts loaded in source files received from the allow listed domains, ignoring all other scripts.

Co-authored-by: Jakub Miarka <jakub.miarka@digital.cabinet-office.gov.uk>

### CODE REVIEW

File: config/initializers/content_security_policy.rb [hunk 1]
```diff
@@ -4,17 +4,17 @@
...
-# Rails.application.config.content_security_policy do |policy|
-#   policy.default_src :self, :https
-#   policy.font_src    :self, :https, :data
-#   policy.img_src     :self, :https, :data
-#   policy.object_src  :none
-#   policy.script_src  :self, :https
-#   policy.style_src   :self, :https
+Rails.application.config.content_security_policy do |policy|
+  policy.default_src :self
+  policy.font_src    :self, ENV["ASSET_HOST"]
```
Comment (jakubmiarka) [inline config/initializers/content_security_policy.rb]: I think Travis doesn't like nil value for the ENV

Comment (Rosa-Fox) [reply config/initializers/content_security_policy.rb]: @jakubmiarka I have amended to only set the csp header to only be set for prod. This means that we don't need to have the env var set in development. There is a bit of a risk that we could add images from a different source or scripts (maybe analytics for example) in dev and not realise that they are blocked until we go to prod... but I don't think the likelihood of us doing so is very high?

Comment (jakubmiarka) [reply config/initializers/content_security_policy.rb]: yep, that's fine.. staging is treated as prod as well, so we would probably catch that in there too.. and we shouldn't be really adding anything external, and if we're we'll have to keep this in mind ;)

Comment (Rosa-Fox) [reply config/initializers/content_security_policy.rb]: 👍

**Pull request merged**